introduction.md

```diff
@@ -0,0 +1 @@
+This is file called as "introduction.md" in dojo_rules repository.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

Reply:
Database,JavaScript, HTML & CSS